packages/ingest/src/types.ts

```diff
@@ -0,0 +1,26 @@
+export interface ProviderClientListItem {
```

Comment:
What does this do..?

Reply:
It's making sure that the providers all provide the same expected interface (once they're each migrated to TS).

Reply:
Each of the providers' `list` method is returning objects with different fields. This is meant to specify the minimum fields that all of the providers provide. It'll get fleshed out more as additional providers are migrated to TS.